packages/website/src/Events/Input/Messages/BotMessageDoc.md

```diff
@@ -0,0 +1,44 @@
+## BotMessage
+
+The `BotMessage` component stores messages generated by an automated assistant within a conversation.
+
+---
+
+| Reference | Link |
+| --- | --- |
+| Encre Concept | [Serializable, ContentLike, MessageRole, BaseMessageFields](**-a-link-to-the-corresponding-concept-documentation-**) |
+| Encre Node | [Bot Message Node](**-a-link-to-the-corresponding-node-documentation-**) |
+| API | [** A reference to API Documentation **](**-a-link-to-the-corresponding-api-documentation-**) |
+
+### Overview
+
+The `BotMessage` component is crucial for handling messages generated by an automated assistant (e.g., chatbots) in a conversation. It inherits from the `BaseMessage` class and provides additional methods specific to bot messages.
```

Comment:
I believe this is directly copied from the API documentation. You probably need to write a more user-friendly sentence to explain the usage of the BotMessage. I'm not very sure to say where it is inherited from since the common user don't need to know that.

In addition, it is necessary to explain where it could be used to. For example, the messages are usually used as the outputs from the Chat Models such as OpenAIChat, etcs.